lib/facts.ts

```diff
@@ -62,6 +62,11 @@ export interface TestedFilesFact {
   data: string[];
 }
 
+export interface ApplicationFilesFact {
+  type: "applicationFiles";
+  data: string[];
```

Comment:
Create an interface like `ApplicationFileInfo` to allow more flexibility across different ecosystems